assets/src/edit-story/elements/video/output.js

```diff
@@ -31,6 +31,7 @@ function VideoOutput({ element, box }) {
     autoPlay: 'autoplay',
     poster: resource.poster,
     layout: 'fill',
+    loop: element.loop,
```

Comment:
`loop` could be added to `const { resource } = element;` above instead.

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/653/commits/48c0e647545c59f36df32b549a995738f283563e